src/cljc/rems/common/application_util.cljc

```diff
@@ -27,3 +29,26 @@
 
 (defn is-handler? [application user]
   (contains? (workflow-handlers application) user))
+
+(defn- str-to-int [s]
+  #?(:clj (Integer/parseInt s)
+     :cljs (js/parseInt s 10)))
+
+;; (defn- printToConsole [s]
+;;   #?(:clj (println s)
+;;      :cljs (js/console.log s)))
+
+(defn parse-sortable-external-id
+  "The idea is to parse all numbers from the string to vector and then compare the values in the vector"
+  [external-id]
+  (when external-id
+    (when-let [number-sequence  (seq (re-seq #"\d+" external-id))]
+      (vec (map str-to-int
+                number-sequence)))))
+
+(deftest test-parse-sortable-external-id
+  (is (= [2020 10] (parse-sortable-external-id "2020/10")))
+  (is (= [54 14 01 00 2002 21] (parse-sortable-external-id "THL/54/14.01.00/2002-rems/21")))
```

Comment:
This should not lose "THL" or "-rems" since those can be used in the comparison ...